### PR TITLE
Get info about match and players

### LINKED ISSUE #3: Get info about match and players
- [x] Add OpenDotaApi gem
- [x] Fetch info for the match
- [x] Create match entity
- [x] Create player entity
- [x] Get info about all players from match
- [x] Save match to DB
- [x] Save players to DB

### LINKED ISSUE #5: Calculate FP based on match info
- [x] Create FPCalculator entity
- [x] Create method for calculating FP

### LINKED ISSUE #7: Display table with players results for match

### PR DESCRIPTION
**Pull request by OrestOs**
close #3 
close #5 
close #7

### CODE REVIEW

**Comment by AndriiAkulov** [comment]
@vergilet Fixed

File: app/controllers/player_match_statistics_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class PlayerMatchStatisticsController < ApplicationController
+  def index
+    @match = Match.find_by_od_match_id(params[:match_id])
+    @statistics = PlayerMatchStatistic.where(match_id: @match.id)
+
+    @players_fp = @statistics.map do |statistic|
```
Comment (vergilet) [inline app/controllers/player_match_statistics_controller.rb]: We are going to implement presenter here  - `Matches::StatisticPresenter`

File: app/controllers/player_match_statistics_controller.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class PlayerMatchStatisticsController < ApplicationController
```
Comment (vergilet) [inline app/controllers/player_match_statistics_controller.rb]: As far this is nested controller, you should use namespaces:
```ruby
class Matches::StatisticsController < ApplicationController
```

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,12 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  # get 'matches/find/:id'
+
+  resources :matches do
+    resources :player_match_statistics
```
Comment (vergilet) [inline config/routes.rb]:
```ruby 
resources :player_match_statistics, controller: 'matches/statistics', only: [:index]
```
# maybe you need to use `show` action here ?

Comment (vergilet) [inline config/routes.rb]: ok, it can be index for now.

**Pull request merged**